R/main.R

```diff
@@ -21,15 +21,13 @@ Options:
        queue_id = dat$queue_id,
        workers = as.integer(dat$workers),
        results_dir = validate_path(dat[["results_dir"]]),
-       prerun_dir = validate_path(dat[["prerun_dir"]]),
        inputs_dir = validate_path(dat[["inputs_dir"]]))
 }
 
 main_api <- function(args = commandArgs(TRUE)) {
   # nocov start
   dat <- main_api_args(args)
-  api <- api(dat$queue_id, dat$workers, dat$results_dir, dat$prerun_dir,
-             dat$inputs_dir)
+  api <- api(dat$queue_id, dat$workers, dat$results_dir, dat$inputs_dir)
```

Comment:
above in the docopt string, should prerun-dir go (with deletions also in the tests aftwards, possibly affecting deploy?)